### PR TITLE
gitian-keys: Add signer aliases, some historical keys

### PR DESCRIPTION
**Pull request by laanwj**
Update `keys.txt` for `gitian-verify.py`  (bitcoin-core/bitcoin-maintainer-tools#71)

- Extend line format with signer aliases in parentheses
- Add a few historical signers
- Sort the file by name

Co-authored-by: fanquake <fanquake@gmail.com>

### CODE REVIEW

File: contrib/gitian-keys/keys.txt [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
-32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC Jonas Schnelli
+82921A4B88FD454B7EB8CE3C796C4109063D4EAF Jon Atack (jonatack)
+32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC Jonas Schnelli (jonasschnelli)
 4B4E840451149DD7FB0D633477DFAB5C3108B9A8 Jorge Timon
 C42AFF7C61B3E44A1454CD3557AF762DB3353322 Karl-Johan Alm (kallewoof)
-E463A93F5F3117EEDE6C7316BD02942421F4889F Luke Dashjr
-B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B Marco Falke
+30DE693AE0DE9E37B3E7EB6BBFF0F67810C1EED1 Lisa Neigut (niftynei)
+E463A93F5F3117EEDE6C7316BD02942421F4889F Luke Dashjr (luke-jr)
+B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B Marco Falke (marco)
```
Comment (jonatack) [inline contrib/gitian-keys/keys.txt:30]: MarcoFalke?

File: contrib/gitian-keys/keys.txt [hunk 2]
```diff
@@ -1,41 +1,51 @@
...
 D62A803E27E7F43486035ADBBCD04D8E9CCCAC2A Paul Rabahy
 37EC7D7B0A217CDB4B4E007E7FAB114267E4FA04 Peter Todd
-D762373D24904A3E42F33B08B9A408E71DAAC974 Pieter Wuille (Location: Leuven, Belgium)
-133EAC179436F14A5CF1B794860FEB804E669320 Pieter Wuille
+D762373D24904A3E42F33B08B9A408E71DAAC974 Pieter Wuille [Location: Leuven, Belgium] (sipa)
+133EAC179436F14A5CF1B794860FEB804E669320 Pieter Wuille (sipa)
 6A8F9C266528E25AEB1D7731C2371D91CB716EA7 Sebastian Falbesoner (theStack)
 A8FC55F3B04BA3146F3492E79303B33A305224CB Sebastian Kung (TheCharlatan)
-ED9BDF7AD6A55E232E84524257FF9BDBCC301009 Sjors Provoost
+ED9BDF7AD6A55E232E84524257FF9BDBCC301009 Sjors Provoost (sjors)
```
Comment (jonatack) [inline contrib/gitian-keys/keys.txt:45]: Sjors?

**Review by jonatack — COMMENTED** [review]
Concept ACK.

File: contrib/gitian-keys/keys.txt [hunk 3]
```diff
@@ -1,41 +1,51 @@
...
-152812300785C96444D3334D17565732E08E5E41 Andrew Chow
+152812300785C96444D3334D17565732E08E5E41 Andrew Chow (achow101)
 590B7292695AFFA5B672CBB2E13FC145CD3F4304 Antoine Poinsot (darosior)
 0AD83877C1F0CD1EE9BD660AD7CC770B81FD22A8 Ben Carman (benthecarman)
 912FD3228387123DC97E0E57D5566241A0295FA9 BtcDrak
 C519EBCF3B926298946783EFF6430754120EC2F4 Christian Decker (cdecker)
+18AE2F798E0D239755DA4FD24B79F986CBDF8736 Chun Kuan Le (ken2812221)
 F20F56EF6A067F70E8A5C99FFF95FAA971697405 centaur
-C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields
+C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields (cfields)
```
Comment (jonatack) [inline contrib/gitian-keys/keys.txt:11]: maybe
```suggestion
C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields (cfields, theuni)
```

File: contrib/gitian-keys/keys.txt [hunk 4]
```diff
@@ -1,41 +1,51 @@
...
 BF6273FAEF7CC0BA1F562E50989F6B3048A116B5 Dev Random
-6D3170C1DC2C6FD0AEEBCA6743811D1A26623924 Douglas Roark
+6D3170C1DC2C6FD0AEEBCA6743811D1A26623924 Douglas Roark (droark)
+1C6621605EC50319C463D56C7F81D87985D61612 Emanuele Cisbani (cisba)
 9A1689B60D1B3CCE9262307A2F40A9BF167FBA47 Erik Mossberg (erkmos)
 D35176BE9264832E4ACA8986BF0792FBE95DC863 fivepiece
+6F993B250557E7B016ADE5713BDCDA2D87A881D9 Fuzzbawls (Fuzzbawls)
```
Comment (jonatack) [inline contrib/gitian-keys/keys.txt:17]: seems redundant

File: contrib/gitian-keys/keys.txt [hunk 5]
```diff
@@ -1,41 +1,51 @@
...
 01CDF4627A3B88AAE4A571C87588242FBE38D3A8 Gavin Andresen
```
Comment (jonatack) [inline contrib/gitian-keys/keys.txt:18]:
```suggestion
01CDF4627A3B88AAE4A571C87588242FBE38D3A8 Gavin Andresen (gavinandresen)
```

File: contrib/gitian-keys/keys.txt [hunk 6]
```diff
@@ -1,41 +1,51 @@
...
 D1DBF2C4B96F2DEBF4C16654410108112E7EA81F Hennadii Stepanov (hebasto)
+A2FD494D0021AA9B4FA58F759102B7AE654A4A5A Ilyas Ridhuan (IlyasRidhuan)
+D3F22A3A4C366C2DCB66D3722DA9C5A7FA81EA35 Jarol Rodriguez (jarolrod)
+7480909378D544EA6B6DCEB7535B12980BB8A4D3 Jeffri H Frontz (jhfrontz)
 D3CC177286005BB8FF673294C5242A1AB3936517 jl2012
-82921A4B88FD454B7EB8CE3C796C4109063D4EAF Jon Atack
-32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC Jonas Schnelli
+82921A4B88FD454B7EB8CE3C796C4109063D4EAF Jon Atack (jonatack)
+32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC Jonas Schnelli (jonasschnelli)
 4B4E840451149DD7FB0D633477DFAB5C3108B9A8 Jorge Timon
```
Comment (jonatack) [inline contrib/gitian-keys/keys.txt:26]:
```suggestion
4B4E840451149DD7FB0D633477DFAB5C3108B9A8 Jorge Timon (jtimon)
```

File: contrib/gitian-keys/keys.txt [hunk 7]
```diff
@@ -1,41 +1,51 @@
...
 C42AFF7C61B3E44A1454CD3557AF762DB3353322 Karl-Johan Alm (kallewoof)
-E463A93F5F3117EEDE6C7316BD02942421F4889F Luke Dashjr
-B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B Marco Falke
+30DE693AE0DE9E37B3E7EB6BBFF0F67810C1EED1 Lisa Neigut (niftynei)
+E463A93F5F3117EEDE6C7316BD02942421F4889F Luke Dashjr (luke-jr)
+B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B Marco Falke (marco)
 07DF3E57A548CCFB7530709189BBB8663E2E65CE Matt Corallo (BlueMatt)
```
Comment (jonatack) [inline contrib/gitian-keys/keys.txt:31]: maybe, per https://github.com/bitcoin-core/gitian.sigs/commits?author=TheBlueMatt

```suggestion
07DF3E57A548CCFB7530709189BBB8663E2E65CE Matt Corallo (TheBlueMatt)
```

**Comment by practicalswift** [comment]
Concept ACK

File: contrib/gitian-keys/keys.txt (hunk 4 above)
Comment (laanwj) [reply contrib/gitian-keys/keys.txt:17]: It's not. The script ignores the "real name" (the first part) only looks at the aliases in parentheses.

File: contrib/gitian-keys/keys.txt (hunk 7 above)
Comment (laanwj) [reply contrib/gitian-keys/keys.txt:31]: That would be if he used that as directory name in `gitian.sigs`, but he used BlueMatt. It's not the github name, it's the signer alias as used for gitian.

File: contrib/gitian-keys/keys.txt (hunk 5 above)
Comment (laanwj) [reply contrib/gitian-keys/keys.txt:18]: Sure, but did he ever do a gitian build?

File: contrib/gitian-keys/keys.txt (hunk 3 above)
Comment (laanwj) [reply contrib/gitian-keys/keys.txt:11]:
```
gitian.sigs$ git ls-files|grep -i /theuni/
[nothing]
```

File: contrib/gitian-keys/keys.txt (hunk 6 above)
Comment (laanwj) [reply contrib/gitian-keys/keys.txt:26]: ACK, assuming he used the same key back then
```
$ git ls-files|grep -i /jtimon/
0.13.2-linux/jtimon/bitcoin-linux-0.13-build.assert
0.13.2-linux/jtimon/bitcoin-linux-0.13-build.assert.sig
0.13.2-win-unsigned/jtimon/bitcoin-win-0.13-build.assert
0.13.2-win-unsigned/jtimon/bitcoin-win-0.13-build.assert.sig
```

**Comment by laanwj** [comment]
Updated per @jonatack's comments to add signer aliases for everyone I could find the directory name in `gitian.sigs` for.

**Comment by laanwj** [comment]
Updated to remove a key that was not used to sign any build result in `gitian.sigs`.

**Review by Sjors — APPROVED** [review]
Concept ACK

The signer aliases in 9d830c632f0be78f44f11871dd54e3babeb8dc4f look reasonable (my own is correct), but I didn't thoroughly check them.

**Comment by laanwj** [comment]
Rebased for #20989

**Pull request merged**